core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -359,102 +359,103 @@ class ParallelScan<FunctorType, Kokkos::RangePolicy<Traits...>,
   using ValueJoin   = Kokkos::Impl::FunctorValueJoin<FunctorType, WorkTag>;
   using ValueOps    = Kokkos::Impl::FunctorValueOps<FunctorType, WorkTag>;
 
+  using value_type     = typename ValueTraits::value_type;
   using pointer_type   = typename ValueTraits::pointer_type;
   using reference_type = typename ValueTraits::reference_type;
 
   const FunctorType m_functor;
   const Policy m_policy;
-  /*
-    template< class TagType >
-    inline static
-    typename std::enable_if< std::is_same< TagType , void >::value >::type
-    exec_range( const FunctorType & functor
-              , const Member ibeg , const Member iend
-              , reference_type update , const bool final )
```

Comment:
Note: all the removed code is irrelevant. That was dead (commented code) from the original CPU OpenMP backend).